components/board/digital_interrupts.go

```diff
@@ -133,7 +141,7 @@ func (i *BasicDigitalInterrupt) Ticks(ctx context.Context, num int, now uint64)
 }
 
 // Tick records an interrupt and notifies any interested callbacks.
-func (i *BasicDigitalInterrupt) Tick(ctx context.Context, high bool, not uint64) error {
+func (i *BasicDigitalInterrupt) Tick(ctx context.Context, high bool, nanoseconds uint64) error {
```

Comment:
Mild preference to either rename this or add another comment. Is `nanoseconds` when the tick occurred, or the time since the last tick, or something else?

...I guess I can't think of a second meaning for it, so maybe it's fine as-is. but at first glance, I wasn't sure what it would represent.